Fulfillment/APIs/In progress orders.md

```diff
@@ -28,28 +28,44 @@ Example: https://demo-oms.hotwax.io/api/solr-query
       "start": 0
     },
     "query": "(*:*)",
-    "filter": "docType: OISGIR AND picklistItemStatusId: PICKITEM_PENDING AND -fulfillmentStatus: Rejected AND -shipmentMethodTypeId: STOREPICKUP AND facilityId: Store_1 AND productStoreId: STORE"
+    "filter": "docType: OISGIR AND " +
+              "picklistItemStatusId: PICKITEM_PENDING AND " +
```

Comment:
format does not match outstanding orders query

Reply:
changed the format, please have a look.